README.md

```diff
@@ -196,45 +199,45 @@ To migrate a LoopBack 2.x application to use `strong-error-handler`:
 
 1. In `package.json` dependencies, remove `"errorhandler": "^x.x.x”,`
 1. Install the new error handler by entering the command:
-    <pre>npm install --save strong-error-handler</pre>
+<pre>npm install --save strong-error-handler</pre>
 1. In `server/config.json`, remove:
-    <pre>
-    "remoting": {
-      ...
-      "errorHandler": {
-        "disableStackTrace": false
-      }</pre>
-    and replace it with:
-    <pre>
-    "remoting": {
-      ...,
-      "rest": {
-        "handleErrors": false
-      }</pre>
+   <pre>
+   "remoting": {
+     ...
+     "errorHandler": {
+       "disableStackTrace": false
+     }</pre>
+   and replace it with:
+   <pre>
+   "remoting": {
+     ...,
+     "rest": {
+       "handleErrors": false
+     }</pre>
 1. In `server/middleware.json`, remove:
-    <pre>
-    "final:after": {
-      "loopback#errorHandler": {}
-    }</pre>
-    and replace it with:
-    <pre>
-    "final:after": {
-      "strong-error-handler": {}
-    }</pre>
+   <pre>
+   "final:after": {
+     "loopback#errorHandler": {}
+   }</pre>
+   and replace it with:
+   <pre>
+   "final:after": {
+     "strong-error-handler": {}
+   }</pre>
 1. Delete `server/middleware.production.json`.
 1. Create `server/middleware.development.json` containing:
-    <pre>
-    "final:after": {
-      "strong-error-handler": {
-        "params": {
-          "debug": true,
-          "log": true
-        }
-      }
-    }
+<pre>
+"final:after": {
+"strong-error-handler": {
+"params": {
+"debug": true,
+"log": true
+}
+}
+}
```

Comment:
Unrelated changes, please revert them. 

Also note that you are breaking indentation of the pre-formatted text.